pkg/omf/omf.fish

```diff
@@ -119,13 +123,18 @@ function omf -d "Oh My Fish"
 
     case "t" "theme"
       if test (count $argv) -eq 1
-        set -l ostype (uname)
         set -l theme (cat $OMF_CONFIG/theme)
-        set -l regex "[[:<:]]($theme)[[:>:]]"
-        test "$ostype" != "Darwin"; and set regex "\b($theme)\b"
-
-        omf.packages.list --database --theme | column | sed -E "s/$regex/"(omf::em)"\1"(omf::off)"/"
-        omf::off
+        set -l highlight_current "(^|\s)($theme)(\s|\$)"
```

Comment:
The order doesn't change anything. What I have seen is that there is no '\s' for osx. We would have to use a similar regex we used to that was exclusive for osx.

I think documenting is a good start! Could you update the documentation?